bin/mprocs.yaml

```diff
@@ -22,5 +22,9 @@ procs:
         # docker-compose makes sure the stack is up, and then follows its logs - but doesn't tear down on exit for speed
         shell: 'docker compose -f docker-compose.dev.yml up -d && docker compose -f docker-compose.dev.yml logs --tail=0 -f'
 
+    storybook:
+        shell: 'pnpm storybook'
+        autostart: false
```

Comment:
Sneaky, but the down-by-default Storybook makes it so convenient to start Storybook when needed